docs/reference/hip_runtime_api/modules_list.rst

```diff
@@ -0,0 +1,28 @@
+* :ref:`driver_reference`
```

Comment:
Is there any way to indicate which files these modules can be found in?

Reply:
Updated.